packages/ds-healthcare-gov/src/components/Header/Header.jsx

```diff
@@ -247,6 +248,10 @@ _Header.propTypes = {
    * the `defaultMenuLinks` method or the links provided by the `links` prop.
    */
   submenuBottom: PropTypes.node,
+  /**
+   * Element added to display content on Header bottom section
+   */
+  headerBottom: PropTypes.node,
```

Comment:
We've got some intermediate Typescript typing going on right now, so any time a prop is added, we also have to update the type definition file. Could you please update [this file](https://github.com/CMSgov/design-system/blob/ff52e04552042f02931fc93359325b93616e4f1d/packages/ds-healthcare-gov/src/types/Header/index.d.ts) with

```
  headerBottom?: React.ReactNode;
```
in the `HeaderProps` interface

Reply:
Done